src/disk91_LoRaE5.cpp

```diff
@@ -169,29 +169,40 @@ bool Disk91_LoRaE5::readOneByte(uint8_t adr, uint8_t * v) {
 // Persistance
 struct s_persist {
     uint8_t version;
-    uint8_t deveui[8];
-    uint8_t appeui[8];
-    uint8_t appkey[16];
+    // uint8_t deveui[8];
+    // uint8_t appeui[8];
+    // uint8_t appkey[16];
+    char deveui[16];
```

Comment:
No reason for this and as you can see it has an impact on the memory footprint x2. Something impacting the NVM, Flash and RAM.